Formula/w/wasmedge.rb

```diff
@@ -2,12 +2,11 @@ class Wasmedge < Formula
   desc "Lightweight, high-performance, and extensible WebAssembly runtime"
   homepage "https://WasmEdge.org/"
   license "Apache-2.0"
-  revision 2
   head "https://github.com/WasmEdge/WasmEdge.git", branch: "master"
 
   stable do
```

Comment:
I think the patch and stable block can be removed.

Reply:
👍